e2e/tests/edusharing.ts

```diff
@@ -12,16 +12,16 @@ Scenario('The editor can be called via the LTI Workflow', ({ I }) => {
   expectEditorOpenedSuccessfully(I)
 })
 
-Scenario(
-  'Fails when the LTI custom claim (sent by edusharing) is missing a non-optional property',
-  ({ I }) => {
-    I.removePropertyInCustom('dataToken')
+// Scenario(
```

Comment:
If you think it will not be used again, you can just remove the code, it is better than having some commented code hanging around.
Otherwise, you can Scenario.skip it

Reply:
Good point! I removed it but we can still find it in this pull request if we need it later.